docs/arithmetic_expression.rst

```diff
@@ -45,6 +45,36 @@ The associativity and precedence of the overloaded Python arithmetic operators a
  * :code:`Int(1) + Int(2) + Int(3)` is equivalent to :code:`Add(Add(Int(1), Int(2)), Int(3))`
  * :code:`Int(1) + Int(2) * Int(3)` is equivalent to :code:`Add(Int(1), Mul(Int(2), Int(3)))` 
 
+Byteslice Arithmetic
+~~~~~~~~~~~~~~~~~~~~
+
+Byteslice arithemetic is available for Teal V4 and above. 
+In PyTeal, byteslice arithmetic expressions include :code:`TealType.Bytes` values as arguments (with the exception of :code:`BZero`).
+The table below summarizes the byteslize arithmetic operations in PyTeal.
+
+=================================== ================================================= ===========================
+Operator                            Semantics                                         Example
```

Comment:
I think the `Semantics` column is a little misleading since we don't provide overloads for these operations. Instead, it would probably be more useful to provide the produced values for the examples.

Reply:
Agreed, changing the docs columns to:
`Operator` | `Return Type` | `Example` | `Example Result`